src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs

```diff
@@ -123,7 +123,7 @@ public override bool Equals(object obj)
                 return false;
             }
 
-            // RFC2616: 14.20: unquoted tokens should use case-INsensitive comparison; quoted-strings should use
+            // RFC2616: 14.20: unquoted tokens should use case-Insensitive comparison; quoted-strings should use
```

Comment:
This one looks intentional

Reply:
looking at the spec, it always uses `case-insensitive`. so not sure why the comment would diverge from that?

Reply:
emphasis